docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md

```diff
@@ -0,0 +1,1069 @@
+
+* [Ktor](https://ktor.io/docs/create-client.html) as an HTTP client for retrieving data over the internet.
+* [`kotlinx.serialization`](https://github.com/Kotlin/kotlinx.serialization) to deserialize JSON responses into objects
+  of entity classes
+* [`kotlinx.coroutines`](https://github.com/Kotlin/kotlinx.coroutines) to write asynchronous code
+* [SQLDelight](https://github.com/cashapp/sqldelight) to generate the Kotlin code from SQL queries to create a type-safe
+  database API
+
+> You can find the [template project](https://github.com/kotlin-hands-on/kmm-networking-and-data-storage) as well as the
+> source code of the [final application](https://github.com/kotlin-hands-on/kmm-networking-and-data-storage/tree/final) on
+> the corresponding GitHub repository.
+>
+{type="note"}
+
+## Before you start
+
+1. Download and install [Android Studio](https://developer.android.com/studio/).
+2. Search for
+   the [Kotlin Multiplatform Mobile plugin](https://plugins.jetbrains.com/plugin/14936-kotlin-multiplatform-mobile)
+   in the Android Studio Marketplace and install it for working with the shared module and Android project.
+3. Download and install [Xcode](https://developer.apple.com/xcode/) for building the iOS application.
+
+For more details, see the [Set up the environment](multiplatform-mobile-setup.md) section.
+
+## Create a Multiplatform project
+
+1. In Android Studio, select **File** | **New** | **New Project**. In the list of project templates, select **Kotlin
+   Multiplatform App** and click **Next**.
+
+   ![Kotlin Multiplatform Mobile plugin wizard](multiplatform-mobile-project-wizard-1.png){width=700}
+
+2. Name your application and click **Next**.
+3. Select **Regular framework** in the list of **iOS framework distribution** options.
+
+   ![Kotlin Multiplatform Mobile plugin wizard. Final step](multiplatform-mobile-project-wizard-3.png){width=700}
+
+4. Keep all other options default. Click **Finish**.
+5. To view the complete structure of the multiplatform mobile project, switch the view from **Android** to **Project**.
+
+   ![Project view](select-project-view.png){width=200}
+
+For more on project features and how to use them,
+see [Understand the project structure](multiplatform-mobile-understand-project-structure.md).
+
+> You can find the configured project on the `master` branch [here](https://github.com/kotlin-hands-on/kmm-networking-and-data-storage)
+>
+{type="note"}
+
+## Add dependencies to the multiplatform library
+
+To add a multiplatform library to the shared module, you need to add dependency instructions (`implementation`) for all
+libraries to the `dependencies` block of the relevant source sets in the `build.gradle.kts` file.
+
+Also, both `kotlinx.serialization` and SQLDelight libraries require additional configurations.
+
+1. In the `shared` directory, specify dependencies on all required libraries in the `build.gradle.kts` file:
+
+    ```kotlin
+    val coroutinesVersion = "%coroutinesVersion%"
```

Comment:
Values in the `final` branch are:
val ktorVersion = "2.0.2"
val coroutinesVersion = "1.6.2"

and they differ from current %coroutinesVersion% and %ktorVersion%:
val coroutinesVersion = "1.6.4"
val ktorVersion = "2.1.2"

Reply:
I think it's inevitable. The source project is not updated as often as we update variables for the whole documentation. We have the same situation throughout our tutorials